alacritty/src/event.rs

```diff
@@ -749,6 +749,15 @@ impl<N: Notify + OnResize> Processor<N> {
     {
         let mut scheduler = Scheduler::new();
 
+        if self.config.cursor.style.blinking {
+            if let Err(err) = event_loop
+                .create_proxy()
+                .send_event(Event::TerminalEvent(TerminalEvent::CursorStartBlinking))
+            {
+                warn!("Could not make the cursor blink at startup: {}", err);
+            }
+        }
```

Comment:
The send_event here doesn't need any error handling, just how it's done for other channels in Alacritty.

It also seems pointless to send an event to the event loop, when we are literally right in the event handler, that just seems like a pointless indirection. Can't we just start the blinking ourselves?

Reply:
That sounds like a good idea anyways, since we want to keep our event handler functionally simple and delegate to methods for anything even slightly complex.